packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -449,6 +345,57 @@ const EDiscovery = WebexPlugin.extend({
     return '';
   },
 
+  _isUrl(string) {
+    try {
+      const validUrl = new URL(string);
+
+      return validUrl !== undefined;
```

Comment:
Looked at using require('url') as Adam mentioned. But it took me by surprise as

    console.log(url.parse('16bf0d01-b1f7-483b-89a2-915144158fb9'))

generated

    Url {
      protocol: null,
      slashes: null,
      auth: null,
      host: null,
      port: null,
      hostname: null,
      hash: null,
      search: null,
      query: null,
      pathname: '16bf0d01-b1f7-483b-89a2-915144158fb9',
      path: '16bf0d01-b1f7-483b-89a2-915144158fb9',
      href: '16bf0d01-b1f7-483b-89a2-915144158fb9' }

Felt that URL was more direct as it throws a constructor exception. Which I've gotten used to but I recall that seem unusual originally.

    validUrl !== undefined;

To calm eslint :(. It does not like 

    new URL(string)